Cargo.toml

```
@@ -14,6 +14,9 @@ license = "Apache-2.0"
 edition = "2021"
 rust-version = "1.70"
 
+[package.metadata.docs.rs]
+all-features = true
```

then we should also add this

```suggestion
[package.metadata.docs.rs]
rustdoc-args = ["--cfg", "docsrs"]
all-features = true
```

---

It enables the automatic documentation feature.

---

According to https://github.com/rust-lang/docs.rs/pull/2390 this should already be enabled by default.

---

oh wow guess ive been adding this since way before that was made default. thanks for the link :)